docker/Dockerfile

```diff
@@ -25,23 +27,30 @@ RUN chown -R espressopp:espressopp /home/espressopp/espressopp /home/espressopp/
 USER espressopp
 
 WORKDIR /home/espressopp/espressopp
-RUN mkdir build
+RUN maintainer/check_codestyle.sh ${CODESTYLE} ${TRAVIS_BRANCH} ${TRAVIS_PULL_REQUEST} ${TRAVIS_COMMIT} ${TRAVIS_COMMIT_RANGE}
```

Comment:
Could we do the `${CODESTYLE}" = ON ` check outside the script, that might make it more clear. Plus we don't have to hand `${CODESTYLE}` inside the script.